doc/source/cluster/running-applications/job-submission/openapi.yml

```diff
@@ -0,0 +1,471 @@
+openapi: 3.1.0
+info:
+  title: Ray Jobs API
+  version: 4.0.0
+  description: |
+    This is the specification for the Ray Jobs REST API.
+    See the Ray Jobs documentation for details:
+    https://docs.ray.io/en/latest/cluster/running-applications/job-submission/index.html
+
+paths:
+  /api/version:
+    get:
+      summary: |
+        Get Version
+      description: |
+        Get the Ray Jobs API version and the Ray version running on the cluster.
```

Comment:
Should we use possessive here? Ray Jobs' API? If so then change in all description below.
I guess this would be tantamount to asking "Get Python's version" or "Get Python version." 

I think either would work. But a grammar pedantic would suggest otherwise.